src/Common/Core/Twig/Extensions/BaseTwigModifiers.php

```diff
@@ -221,19 +221,23 @@ public static function showBool($status, bool $reverse = false): string
      * Truncate a string
      *    syntax: {{ $string|truncate($max-length, $append-hellip, $closest-word) }}.
      *
-     * @param string $string The string passed from the template.
+     * @param null|string $string The string passed from the template.
      * @param int $length The maximum length of the truncated string.
      * @param bool $useHellip Should a hellip be appended if the length exceeds the requested length?
      * @param bool $closestWord Truncate on exact length or on closest word?
      *
      * @return string
      */
     public static function truncate(
-        string $string,
+        string $string = null,
```

Comment:
wrong fix, now you have an optional parameter before a required parameter, the correct typehint is
`?string $string`

Reply:
Indeed, my bad.